### PR TITLE
Better resize columns in the VolumeSelectionWindow

### LINKED ISSUE #402: Publisher name gets cut-off when the series name is too long
It's mainly the search window that this becomes annoying with but others should probably remember their size and position too.

### PR DESCRIPTION
**Pull request by lordwelch**
Fixes #402

### CODE REVIEW

File: comictaggerlib/volumeselectionwindow.py [hunk 1]
```diff
@@ -474,6 +474,16 @@ def categorize(result: ComicVolume) -> int:
         self.twList.setSortingEnabled(True)
         self.twList.selectRow(0)
         self.twList.resizeColumnsToContents()
+        # Get the width of the issues, year and publisher columns
+        owidth = self.twList.columnWidth(1) + self.twList.columnWidth(2) + self.twList.columnWidth(3)
```
Comment (mizaki) [inline comictaggerlib/volumeselectionwindow.py:480]: There is a margin on the right which is lost on the publisher column. Adding about 15 seems to be about right:
```
owidth = self.twList.columnWidth(1) + self.twList.columnWidth(2) + (self.twList.columnWidth(3) + 15)
```

**Comment by mizaki** [comment]
I would also argue for making the window a bit wider and taller. As it's modal leaving about 10% space on each side would allow for less likelihood of needing two lines for a title.

File: comictaggerlib/volumeselectionwindow.py (hunk 1 above)
Comment (lordwelch) [reply comictaggerlib/volumeselectionwindow.py:480]: Fixed by using:
```python
self.twList.contentsRect().width() - owidth
```

**Comment by mizaki** [comment]
Looks good. Window could still suffer being a little bit bigger though but I won't mention it.

**Pull request merged**